src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    // number of figures available
+    public static final int FIGURE_COUNT = 5;
+    // the maximum length of each segment in the figure
+    public static final int UPPER_BOUND = 20;
+    // the minimum length of each segment in the figure
+    public static final int LOWER_BOUND = 1;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int randomIndex = random.nextInt(FIGURE_COUNT);
+        String randomColor = new ColorSupplier().getRandomColor();
+        switch (randomIndex) {
+            case 0:
```

Comment:
In one of PRs of other student, I proposed a bit different approach - instead of using indexes, you can introduce an enum (named for example `FigureType`) that will hold names of each figure type and use it with `switch` instruction. 
So, the result would be:
```java
FigureType figureType = getRandomFigureType();
switch(figureType) {
    case TRIANGLE:
    // handler for triangle   
    case SQUARE:
    // handler for square etc. etc.
```

That seems to be more natural approach comparing to using artificial identifier for each shape.

Reply:
Yes, indeed that would make the code easier to understand. I'll try to remember it.
Thank you.